integration_test/third_party_apps_data/applications/varnish/metadata.yaml

```diff
@@ -5,12 +5,13 @@ description: |-
   The Varnish integration collects cache and session metrics.
   It monitors the number of objects entering and exiting the cache,
   as well as the number of sessions and backend connections.
-  The integration also collects Varnish access logs and parses them
+  The integration also collects Varnish logs and parses them
   into a standardized json payload.
-
   For more information about Varnish, see
   [Varnish HTTP Cache](https://varnish-cache.org/).
 supported_app_version: ["6.x", "7.0.x"]
+configure_integration: |-
+  The Varnish logging processor processes logs using [varnishncsa](https://varnish-cache.org/docs/6.0/reference/varnishncsa.html). Varnish can enable varnishncsa logging by following [this](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/) guide depending on os. Logs are expected to be in the default format and a logrotation should be setup.
```

Comment:
Can we also specify in here that the Ops Agent by default expects the configuration of Varnish to write to `/var/log/varnish/varnishncsa.log`. If another file is chosen, the user will have to also update the receiver configuration